opensrp-reveal/src/main/java/org/smartregister/reveal/server/JavaHTTPServer.java

```diff
@@ -0,0 +1,143 @@
+    public static String styleJsonFile;
+    static String styleJson;
+    // port to listen connection
+    public static final int PORT = 8783;
+
+    // verbose mode
+    static final boolean verbose = true;
+
+    // Client Connection via Socket Class
+    private Socket connect;
+
+    public JavaHTTPServer(Socket c) {
+        connect = c;
+    }
+
+    public static void main(String[] args) {
+        try {
+            ServerSocket serverConnect = new ServerSocket(PORT);
+            System.out.println("Server started.\nListening for connections on port : " + PORT + " ...\n");
+
+            digitalGlobeIdPlaceHolder = StringUtils.isNotBlank(digitalGlobeIdPlaceHolder) ? digitalGlobeIdPlaceHolder : DEFAULT_DG_ID_PLACEHOLDER;
+            styleJsonFile = StringUtils.isNotBlank(styleJsonFile) ? styleJsonFile : DEFAULT_STYLE_JSON_FILE;
+
+            styleJson = Utils.readAssetContents(RevealApplication.getInstance().getApplicationContext(), styleJsonFile);
+            styleJson = styleJson.replace(digitalGlobeIdPlaceHolder, BuildConfig.DG_CONNECT_ID);
+
+            // we listen until user halts server execution
+            while (true) {
+                JavaHTTPServer myServer = new JavaHTTPServer(serverConnect.accept());
+
+                if (verbose) {
+                    System.out.println("Connecton opened. (" + new Date() + ")");
+                }
+
+                // create dedicated thread to manage the client connection
+                Thread thread = new Thread(myServer);
+                thread.start();
+            }
+
+
+        } catch (IOException e) {
+            System.err.println("Server Connection error : " + e.getMessage());
+        }
+    }
+
+    @Override
+    public void run() {
+        // we manage our particular client connection
+        BufferedReader in = null; PrintWriter out = null; BufferedOutputStream dataOut = null;
+        String fileRequested = null;
+
+        try {
+            // we read characters from the client via input stream on the socket
+            in = new BufferedReader(new InputStreamReader(connect.getInputStream()));
+            // we get character output stream to client (for headers)
+            out = new PrintWriter(connect.getOutputStream());
+            // get binary output stream to client (for requested data)
+            dataOut = new BufferedOutputStream(connect.getOutputStream());
+
+            // get first line of the request from the client
+            String input = in.readLine();
+            // we parse the request with a string tokenizer
+            StringTokenizer parse = new StringTokenizer(input);
+            String method = parse.nextToken().toUpperCase(); // we get the HTTP method of the client
+
+            fileRequested = DEFAULT_STYLE_JSON_FILE;
+
+            int fileLength = styleJson.getBytes().length;
+            String content = getContentType(fileRequested);
+
+            if (method.equals("GET")) { // GET method so we return content
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)